raspberryPI3/ros2_ws/src/imu_tools/imu_filter_madgwick/include/imu_filter_madgwick/imu_filter_ros.h

```diff
@@ -39,6 +39,7 @@
 #include "imu_filter_madgwick/base_node.hpp"
 #include "imu_filter_madgwick/visibility_control.h"
 
+#define FRAME_ID "map"
```

Comment:
There is a ros parameter (that I created :)) in the config file named "imu_frame" which permit to change the frame_id of the IMU. Why didn't you used it ?

Reply:
Resolved